common/v2/features/AddAccount/components/DeterministicWallets.tsx

```diff
@@ -137,7 +141,7 @@ class DeterministicWalletsClass extends React.PureComponent<Props, State> {
 
         <Table
           head={['#', 'Address', network.unit, translateRaw('ACTION_5')]}
```

Comment:
Using `Network`, `unit` is replaced by `baseAsset` which provides the `uuid` of the networks base asset.
Once we have the asset you should be able to use the `ticker` param.